language/sv.json

```diff
@@ -0,0 +1,283 @@
+{
+  "semantics": [
+    {
+      "label": "Media",
+      "fields": [
+        {
+          "label": "Typ",
+          "description": "Valfri media att visa ovanför frågan."
+        },
+        {
+          "label": "Inaktivera zoomning i bild"
+        }
+      ]
+    },
+    {
+      "label": "Beskrivning av uppgift",
+      "description": "Beskriv din uppgift här. Beskrivningen kommer att visas ovanför området där text matas in.",
+      "placeholder": "Sammanfatta denna bok på 500 tecken ..."
+    },
+    {
+      "label": "Hjälptext",
+      "description": "Denna text bör hjälpa användaren att komma igång.",
+      "placeholder": "Denna bok handlar om ..."
+    },
+    {
+      "label": "Exempel på lösning",
+      "description": "Om du vill kan du lägga till ett exempel på lösning som visas efter deltagaren har skapat en text. Den kallas exempel på lösning, eftersom det sannolikt inte bara finns en lösning",
+      "fields": [
+        {
+          "label": "Introduktion",
+          "description": "Du kan om du vill ge dina deltagare några förklaringar till ditt exempel. Förklaringarna kommer bara att visas om du också lägger till ett exempel.",
+          "placeholder": "Kom ihåg att du inte förväntades komma till exakt samma lösning. Detta är bara ett exempel."
+        },
+        {
+          "label": "Text för exempel på lösning",
+          "description": "Deltagaren kommer att se en knapp med \"Visa lösning\" efter att ha skickat in, om du skriver en text här."
+        }
+      ]
+    },
+    {
+      "label": "Nyckelord",
+      "widgets": [
+        {
+          "label": "Standard"
+        }
+      ],
+      "entity": "Nyckelord",
+      "field": {
+        "label": "Nyckelord",
+        "fields": [
+          {
+            "label": "Nyckelord",
+            "description": "Nyckelord eller fras att söka efter. Använd en asterisk '*' som wildcard för en eller flera bokstäver. Använd ett snedstreck '/' i början och slutet för att använda ett reguljärt uttryck (regular expression)."
+          },
+          {
+            "label": "Variationer",
+            "description": "",
+            "entity": "variation",
+            "field": {
+              "label": "Variation i nyckelord"
+            }
+          },
+          {
+            "label": "Poäng, val och feedback",
+            "fields": [
+              {
+                "label": "Poäng",
+                "description": "Poäng som användaren får om hen inkluderar detta nyckelord eller dess alternativ i svaret."
+              },
+              {
+                "label": "Förekomster",
+                "description": "Definiera hur många förekomster av nyckelordet eller dess variationer som ska belönas med poäng."
+              },
+              {
+                "label": "Skiftlägeskänsligt",
+                "description": ""
+              },
+              {
+                "label": "",
```

Comment:
There are a couple of empty fields. They should use the English phrase if the file is translated only partially.